.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run build && git add dist/
```

Comment:
I may be a bit naieve here with husky pre-commit flows. but are we assuming here that `npm install` has already been run?

Reply:
Also, should we run test/lint? 

This will ensure semistandard has been run

Reply:
It’s a great proposal, I’ll include it.

Reply:
The proposal was implemented. I close the thread, please feel free to re-open. Thanks!